app/styles/_variables.scss

```diff
@@ -72,8 +72,6 @@ $color-bg-white-transparent: rgba(255, 255, 255, 0.49);
 
 $color-shadow-dark: rgba(0, 0, 0, 0.2);
 
-$color-text-placeholder-gray-dark: #666;
```

Comment:
This is darker than the color that replaces it. There's a good chance that will cause it to fail accessibility.